src/Microsoft.OData.Client/Common.cs

```diff
@@ -149,6 +149,9 @@ internal static ODataVersion ConvertToODataVersion(ODataProtocolVersion maxProto
                 case ODataProtocolVersion.V4:
                     return ODataVersion.V4;
 
+                case ODataProtocolVersion.V401:

```

Comment:
maybe V4_01 is  better?

Reply:
I used 4_01 originally, but CA complained that underscores in enums was bad practice. Not sure why...

---
In reply to: [150043534](https://github.com/OData/odata.net/pull/989#discussion_r150043534) [](ancestors = 150043534)